src/content/admin/cloud-credentials/aws.mdx

```diff
@@ -29,11 +29,11 @@ To do this, Okteto provides the OIDC endpoint of your cluster, which can be foun
   />
 </p>
 
-Store these values as environment variables, you will use them in the next steps:
+Store the values as environment variables, you will use them in the next steps:
 
 ```bash
-export OIDC_ENDPOINT=https://container.googleapis.com/v1/projects/myProject/locations/us-central1/clusters/myCluster
-export OKTETO_SERVICE_ACCOUNT=system:serviceaccount:okteto:okteto
+export OIDC_ISSUER=https://container.googleapis.com/v1/projects/myProject/locations/us-central1/clusters/myCluster
+export OIDC_SUBJECT=system:serviceaccount:okteto:okteto
```

Comment:
@codyjlandstrom I'm changing these to be aligned with what we call them on the UI